tools/publisher/Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2018"
 license = "Apache-2.0"
 publish = false
 
+[workspace]
+
```

Comment:
Is this intended? It feels weird

Reply:
Yeah. This prevents Cargo from thinking the publisher tool is part of the workspace from the root Cargo.toml. Without this, it fails to build. I didn't want us to have to add the publisher tool to the code generated Cargo.toml every time we did a release.